lib/mix/lib/mix/tasks/new.ex

```diff
@@ -428,6 +429,7 @@ defmodule Mix.Tasks.New do
   #       level: :info,
   #       format: "$date $time [$level] $metadata$message\n",
   #       metadata: [:user_id]
+  #
```

Comment:
We shouldn't have this line in this template.

Reply:
oops